Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM nvidia/cuda:11.3.0-cudnn8-runtime-ubuntu18.04
+
+RUN \
+    apt-get update && \
+    apt-get -y install wget \
+    aria2 \
+    rsync \
+    cmake \
+    build-essential \
+    curl \
+    git \
+    vim \
+    nvidia-cuda-toolkit
+
+# install miniconda
+RUN \
+    curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -o miniconda.sh && \
+    sh miniconda.sh -b -p /tools/miniconda && \
+    rm miniconda.sh && \
+    /tools/miniconda/bin/conda init && \
+    /tools/miniconda/bin/conda clean -a
+ENV PATH "$PATH:/tools/miniconda/bin"
+
+COPY . /openfold
+WORKDIR /openfold
+
+# Create environment
+RUN conda env create -f environment.yaml
+SHELL ["conda", "run", "-n", "openfold-env", "/bin/bash", "-c"]
+RUN pip install -r requirements.txt
+RUN pip install nvidia-pyindex \
+    && pip install nvidia-dllogger
+SHELL ["/bin/sh", "-c"]
+
+#clone mmseqs2 and compile
+WORKDIR /
+RUN git clone https://github.com/soedinglab/MMseqs2.git
+RUN mkdir -p /MMseqs2/build
+WORKDIR /MMseqs2/build
+RUN cmake -DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_INSTALL_PREFIX=. .. ; make ; make install
+WORKDIR /openfold
+
+# Download folding resources
+RUN mkdir -p resources
```

Comment:
You don't want to store data in the container It is ~3.5GB.  It should be on the user to mount this data in the correct location.  We should document what the correct location is though and how to create this data (look at postgres docker container with /var/lib/postgresql/data)